spec/features/audit/lists/list_content_items_spec.rb

```diff
@@ -6,29 +6,29 @@
   end
 
   scenario "List Content Items to Audit" do
-    create(:content_item, title: "item1", six_months_page_views: 10_000, content_id: "content-id")
-    create(:content_item, title: "item2")
+    content_item1 = create(:content_item, six_months_page_views: 10_000, allocated_to: @current_user)
+    content_item2 = create(:content_item, allocated_to: @current_user)
 
     visit audits_path
 
-    expect(page).to have_content("item1")
+    expect(page).to have_content(content_item1.title)
     expect(page).to have_content("10,000")
-    expect(page).to have_link("item1", href: "/content_items/content-id/audit")
+    expect(page).to have_link(content_item1.title, href: "/content_items/#{content_item1.content_id}/audit")
```

Comment:
I'd rather not writing expectations based on fixture methods.

Reply:
First commit is doing lots of things. It is easier to review if this is broken down into smaller steps that describe a story.

Reply:
It would've been impossible to make this an atomic commit without these test fixes - all the tests make now-broken assumptions about the state of the page when landing on it.

Reply:
Yes, tests should accompany code changes. But we could have created a commit per tab. Plus other commit for small refactorings? No need to change it now.  It is usually better to have small commits as the review will be done more safely.